src/main/java/de/dhbw/karlsruhe/students/mailflow/core/domain/email/MailboxExportRepository.java

```diff
@@ -0,0 +1,9 @@
+package de.dhbw.karlsruhe.students.mailflow.core.domain.email;
+
+import de.dhbw.karlsruhe.students.mailflow.core.domain.email.value_objects.ExportableMailbox;
+import de.dhbw.karlsruhe.students.mailflow.core.domain.email.exceptions.ExportMailboxException;
+
+public interface MailboxExportRepository {
```

Comment:
```suggestion
public interface ExportableMailboxRepository {
```

Our naming convetion is more like "Object of the Repository" and then + "Repository". This would fit our naming convention more 👍